core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh

```
@@ -631,6 +625,9 @@ else
         echol "Crane binary already exists"
 	else
 		file_url=$(head -n 1 <<< "${CRANE}")
+
+        check_connection wget $file_url
```

I would like to run all connection tests as prerequisites (close to the beginning, under the `--- Checks ---` section) in order to fail early if not met (to save time).